bot/cogs/help.py

```diff
@@ -427,7 +429,15 @@ def _get_command_params(self, cmd):
 
                     # see if the user can run the command
                     strikeout = ''
-                    can_run = await command.can_run(self._ctx)
+
+                    # Patch to make the !help command work outside of #bot-commands again
+                    # This probably needs a proper rewrite, but this will make it work in
+                    # the mean time.
+                    try:
+                        can_run = await command.can_run(self._ctx)
+                    except InChannelCheckFailure:
```

Comment:
Shouldn't we apply this to all check failures?

Reply:
I think this is the only one that raises an exception to be able to return a custom message to the user. The other checks return `True` or `False`, which was the expected return to assign to `can_run`.

Reply:
But, we may want to think of something in case other checks with exceptions get added.